.github/workflows/snyk-docker.yml

```diff
@@ -0,0 +1,23 @@
+name: Snyk security analysis (Docker)
+on: [push, pull_request]
+jobs:
+  security:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Build a Docker image
+        run: docker build -t aidoskuneen/aidos-node:latest .
+      - name: Run Snyk to check Docker image for vulnerabilities
+        # Snyk can be used to break the build when it detects vulnerabilities.
+        # In this case we want to upload the issues to GitHub Code Scanning.
+        continue-on-error: true
+        uses: snyk/actions/docker@master
+        env:
+          SNYK_TOKEN: ${{ secrets.SNYK_TOKEN }}
+        with:
+          image: aidoskuneen/aidos-node:latest
+          args: --file=Dockerfile
+      - name: Upload result to GitHub Code Scanning
```

Comment:
I think this part is incomplete now;

From the docs I see these steps:
```
    - uses: garethr/snyk-to-sarif@master
    - uses: actions/upload-artifact@v2
      with:
        name: SARIF
        path: snyk.sarif
    - name: Upload the SARIF file
      uses: github/codeql-action/upload-sarif@v1
      with:
        sarif_file: snyk.sarif
```

Reply:
I took the config from the action's example also, so I think it should be fine. It cannot find the file because the previous step is failing, and my theory is that SNYK_TOKEN is not available in PRs. So let's merge and see how it performs in the main repo.